COPY/usr/bin/manageiq-db-ready

```diff
@@ -36,11 +36,22 @@ def manageiq_db_ready?(db_yaml, env = "production")
   end
 end
 
+def manageiq_db_pending_migrations?
```

Comment:
This reads backwards to me. I read the method as asking "are there any migrations pending", but it returns true if there _aren't_ any pending.

Reply:
Oh yeah good catch, how about `manageiq_db_migrations_current?` ?

Reply:
Sounds good